### PR TITLE
Fix vagrant provisioning

### PR DESCRIPTION
**Pull request by Strobotti**
Previous pull request was closed with "looks ok" -comment but not merged :D

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -175,7 +175,7 @@ Vagrant.configure("2") do |config|
   config.vm.provision "shell", path: "#{github_url}/scripts/base_box_optimizations.sh", privileged: true
 
   # Provision PHP
-  config.vm.provision "shell", path: "#{github_url}/scripts/php.sh", args: [php_timezone, hhvm, php_version]
+  config.vm.provision 'shell', inline: "sudo add-apt-repository -y ppa:ondrej/php && sudo apt-get update && sudo apt-get -y install php#{php_version}-fpm php#{php_version}-curl php#{php_version}-mbstring php#{php_version}-xml php#{php_version}-mysql php#{php_version}-gd", run: "once"
```
Comment (franzliedke) [inline Vagrantfile]: Why is this line necessary? What about it is different from their `php.sh` script?

**Comment by tobscure** [comment]
I commented with my approval for the changes, leaving it for @franzliedke to give the final OK and merge... you closed it ;)

**Comment by Strobotti** [comment]
Ah, it must have got closed as I tried to fix my fork, as I used my master branch for pull requests - which now that I look at it was completely idiotic :D

I've not used github for contributing to projects maintained by others before, so bear with me... :(

**Comment by franzliedke** [comment]
All good, it takes practice. ;)

The other PR looks quite good, I had just one minor question.

File: Vagrantfile (hunk 1 above)
Comment (Strobotti) [inline Vagrantfile]: The vaprobash repository seemed to be totally out-of-date and not supporting php-versions above 5.6.

Also running remote content like that doesn't really feel like a good idea, so if I had time I'd rewrite the whole provisioning using Ansible etc, but for now I'd just replace all obsolete parts as needed with inline commands...

**Comment by franzliedke** [comment]
Thanks a lot!
